CODEOWNERS

```diff
@@ -0,0 +1 @@
+/tofu/aws/modules/cluster_nodes @rancher/qa-pit-crew
```

Comment:
we need to make sure any groups added here are part of / have permission to this repo

Reply:
Yep! This team has write access to this repo now 🎉 

As an aside, we will need to add other teams as well when any are added to the CODEOWNERS file here